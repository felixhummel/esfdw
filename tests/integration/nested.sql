SELECT
  my_nested,
  my_nested_string,
  my_nested_integer,
  my_nested_as_json
FROM hello
