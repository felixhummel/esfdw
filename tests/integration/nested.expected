"integer"=>20, "string"=>"bar 2"|bar 2|20|{"integer": 20, "string": "bar 2"}
"integer"=>10, "string"=>"bar 1"|bar 1|10|{"integer": 10, "string": "bar 1"}
